### PR TITLE
🎉 New Source: ActiveCampaign

### PR DESCRIPTION
**Pull request by dainiussa**
## What
The following PR adds the new Low Code CDK-based ActiveCampaign Connector. For more context, please refer to [this issue](https://github.com/airbytehq/connector-contest/issues/167).

## How
This PR uses the Low Code CDK to create a ActiveCampaign connector with the following streams:

* campaigns
* contacts
* lists
* deals
* segments
* forms

## Recommended reading order
1. `spec.yaml`
2. `activecampaign.yaml`
3. schemas

## 🚨 User Impact 🚨
No changes to existing code.

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [x] Documentation updated
    - [x] Connector's `README.md`
    - [x] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</details>

## Tests

<details><summary><strong>Acceptance</strong></summary>

image

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-activecampaign/source_activecampaign/schemas/TODO.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# TODO: Define your stream schemas
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-activecampaign/source_activecampaign/schemas/TODO.md:1]: Remove this file.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
A few comments but the implementation looks great! Thanks for the contribution.

File: airbyte-integrations/connectors/source-activecampaign/integration_tests/catalog.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+  "streams": [
+    {
+      "name": "TODO fix this file",
+      "supported_sync_modes": ["full_refresh", "incremental"],
+      "source_defined_cursor": true,
+      "default_cursor_field": "column1",
+      "json_schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
```
[the remark this answers is not in the document]
Comment (dainiussa) [reply airbyte-integrations/connectors/source-activecampaign/integration_tests/catalog.json]: Removed

File: airbyte-integrations/connectors/source-activecampaign/source_activecampaign/schemas/TODO.md (hunk 1 above)
Comment (dainiussa) [reply airbyte-integrations/connectors/source-activecampaign/source_activecampaign/schemas/TODO.md]: Removed

File: airbyte-integrations/connectors/source-activecampaign/test.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+    json_1 = json.loads(a)
+    
+
+    b = '{ "id": "2", "isDisabled": 1, "title": "Demo Requested" }'
+
+    json_2 = json.loads(b)
+
+    print("="*20)
+    print(set(json_2.keys()).difference(json_1.keys()))
```
[the remark this answers is not in the document]
Comment (dainiussa) [reply airbyte-integrations/connectors/source-activecampaign/test.py]: Removed

File: docs/integrations/sources/activecampaign.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+### Performance considerations
+
+The connector has a rate limit of 5 requests per second per account.
+
+## Getting started
+
+### Requirements
+
+* ActiveCampaign account
+* ActiveCampaign API Key
```
[the remark this answers is not in the document]
Comment (dainiussa) [reply docs/integrations/sources/activecampaign.md]: Added the changelog

File: airbyte-integrations/connectors/source-activecampaign/source_activecampaign/activecampaign.yaml [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+
+  # API Docs: https://developers.activecampaign.com/reference/list-all-campaigns
+  campaigns_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "campaigns"
+      primary_key: "id"
+      path: "/api/3/campaigns"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-activecampaign/source_activecampaign/activecampaign.yaml:50]: @dainiussa why not move `/api/3/` to base url?

File: airbyte-integrations/connectors/source-activecampaign/integration_tests/sample_config.json [hunk 6]
```diff
@@ -0,0 +1,3 @@
+{
+  "fix-me": "TODO"
+}
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-activecampaign/integration_tests/sample_config.json:3]: Please fix this file and give one example (remove secrets of course)

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
A few comments and it is almost ready to go! :)
Ask my review again when all comments are resolved.

**Review by marcosmarxm — APPROVED** [review]
congrats @dainiussa I'll publish and merge later.

**Review by marcosmarxm — APPROVED** [review]

**Pull request merged**